src/function.py

```diff
@@ -80,47 +62,30 @@ def try_delete_function_file(client: CogniteClient, external_id: str):
         logger.info(f"Did delete file {external_id}.")
 
 
-def create_and_wait(
-    client: CogniteClient,
-    name: str,
-    external_id: str,
-    function_path: str,
-    file_id: int,
-    api_key: str,
-):
+def create_and_wait(client: CogniteClient, external_id: str, config: FunctionConfig):
     logger.info(f"Will create function {external_id}")
-    function: Function = client.functions.create(
-        name=name,
+    client.functions.create(
+        name=external_id,
         external_id=external_id,
-        file_id=file_id,
-        api_key=api_key,
-        function_path=function_path,
+        folder=config.folder_path,
```

Comment:
Have you tested that, actually?

Reply:
Works fine locally, but adding a "weird dependency" like the slackclient in this case, breaks it because of... see slack: https://cognitedata.slack.com/archives/C010VCGKBCM/p1602583520075600?thread_ts=1602573111.071800&cid=C010VCGKBCM